index.html

```diff
@@ -1293,7 +1295,7 @@ <h1>Featured Speakers</h1>
 									<div class="hover-state text-center preserve3d">
 										<div class="social-links vertical-align">
 											<a target="default" href="https://pep.foundation/"><i class="icon fa fa-external-link"></i></a>
-
+                                            <a href="https://www.linkedin.com/company/4525407/" target="default"><i class="icon social_linkedin"></i></a>
```

Comment:
Same question

Reply:
@vaibhavsingh97 I searched for their Official Linked In Profile. But I was not able to find them. As they have added their Company/Organization Link for other Social Media, I added Their Company/Organization link.